### DISCUSSION TITLE
Allow users to not exit the program after an action completed successfully

### DISCUSSION THREAD

**Discussion by KygekDev** [ThioJoe/YT-Spammer-Purge#179] [Ideas]
Currently we are forced to exit the program after an action completed successfully, for example, scan and delete spam comments across our entire channel:

```
Press Enter to exit...
```

It is inconvenient to re-open the program every time we want to make another operations, because the program needs to keep loading from scratch.

This feature will be useful if we want to make multiple operations at once, for example, scanning multiple channels. Because we don't need to re-open the program each time we want to scan a channel.

---

Hopefully you all understand my explanation, as my English is not that good. If you don't, I'll try my best to re-explain my idea.

**Comment by ThioJoe** [comment]
Yea it's something I plan to do eventually

**Comment by KygekDev** [comment]
Implemented in version [2.10.0](https://github.com/ThioJoe/YT-Spammer-Purge/releases/tag/v2.10.0).
